### ISSUE TITLE
TextArea error on non-string item values

### ISSUE THREAD

**Issue by shalstvedt** [AskNature/AN#108]
Appears for some reason with item key 'width'

**Comment by ethansmith** [comment]
not quite following.. can you provide an example?

**Comment by shalstvedt** [comment]
Just found in the console. If you load a d.strategy texarea tries to call .replace on an integer (some field called width) with the value 200, so it ends up calling 300.replace(...) and that's not a function of integers so it throws an error.

**Comment by ethansmith** [comment]
ah... that error popped up in the code i was working on last night/this morning, but i thought i'd eliminated it. i think i know what it's from.

**Comment by ethansmith** [comment]
Further diagnosis: 
1) only happens when user is logged in
2) doesn't happen on initial page load .. only after route change
3) happens on all content types

**Comment by shalstvedt** [comment]
So why would a textarea be created from a property called "width"? Is this part of Image?

**Comment by ethansmith** [comment]
Issue popped up at 163880d.

**Comment by ethansmith** [comment]
I switched how client/scripts/components/detail/common/datatable.jsx works.. and that 'fix' seems to be the culprit. FIguring out now...

**Comment by shalstvedt** [comment]
Ah, that's right. You changed DataTable to display every item in an entity... and width: 200 is in generic-defaults.js.

**Comment by shalstvedt** [comment]
So DataTable is trying to initialize a field for that value

**Comment by shalstvedt** [comment]
This seems to prevent editing completely

**Comment by ethansmith** [comment]
Easiest fix for now is to do is remove TextArea component from Datatable component. Long term, I think this is fine; Table View will only be used for reference—not editing. Fixed in 2e2e0c3.

**Comment by shalstvedt** [comment]
Having less TextAreas will ultimately speed up the interface, as well... but I also have a fix:

```
    if(item.replace) {
        item = item.replace(/\\/g, '');
        item = item.replace(/&nbsp;/g, '');
    }
```

**Comment by shalstvedt** [comment]
Removing them sounds good though.

**Comment by ethansmith** [comment]
We should probably put your fix in, anyway.

**Comment by shalstvedt** [comment]
Will do

**Comment by shalstvedt** [comment]
Put that fix in d3b2ae7
